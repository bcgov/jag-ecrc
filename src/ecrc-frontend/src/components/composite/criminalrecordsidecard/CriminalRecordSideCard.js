import React from "react";

import SideCard from "../../base/sideCard/SideCard";

export default function CriminalRecordSideCard() {
  const criminalRecord = {
    heading: "Why do I need to apply and consent to a criminal record check?",
    content: "Learn more about the process",
    type: "blue",
    image: "/images/visit-criminal-record-site.PNG",
    imageLink:
      "https://www2.gov.bc.ca/gov/content/safety/crime-prevention/criminal-record-check"
  };

  return <SideCard key="criminal-record" sideCard={criminalRecord} />;
}
