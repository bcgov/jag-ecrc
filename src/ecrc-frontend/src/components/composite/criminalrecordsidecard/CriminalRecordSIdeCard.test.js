import React from "react";
import { create } from "react-test-renderer";

import CriminalRecordSideCard from "./CriminalRecordSideCard";

describe("CriminalRecordSideCard Component", () => {
  test("Matches the snapshot", () => {
    const criminalRecord = create(<CriminalRecordSideCard />);
    expect(criminalRecord.toJSON()).toMatchSnapshot();
  });
});
