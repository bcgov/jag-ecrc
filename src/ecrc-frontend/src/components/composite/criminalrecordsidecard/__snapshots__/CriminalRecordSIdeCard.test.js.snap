// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CriminalRecordSideCard Component Matches the snapshot 1`] = `
<div
  className="dashboard-spacing"
  style={
    Object {
      "position": "relative",
    }
  }
>
  <div
    className="row"
  >
    <div
      className="col-12"
      style={
        Object {
          "paddingTop": "30px",
          "position": "relative",
        }
      }
    >
      <p />
      <section
        className="submit-container"
        id="blue-section"
      >
        <h2
          className="heading-style"
        >
          Why do I need to apply and consent to a criminal record check?
        </h2>
        <div
          className="submit-content"
        >
          <p>
            Learn more about the process
          </p>
          <a
            href="https://www2.gov.bc.ca/gov/content/safety/crime-prevention/criminal-record-check"
          >
            <img
              alt="imagelink"
              height="65px"
              src="/images/visit-criminal-record-site.PNG"
              width="310px"
            />
          </a>
        </div>
      </section>
    </div>
  </div>
</div>
`;
